serverless/sql-databases/concepts.mdx

```diff
@@ -23,7 +23,80 @@ categories:
 </Concept>
 
 <Concept opened>
- ## Credentials
+## Backup
 
- Credentials let you connect to your database via a secret key.
-</Concept>
\ No newline at end of file
+A database backup is a copy of the data stored in the database (including data structure, such as tables). Database backups in Serverless SQL Database are logical backups (as opposed to raw disk volume backups, also called "snapshots"). Database backups can be restored in the current database or any other PostgreSQL database and allows to recover from database crash, data corruption, or data loss.
+</Concept>
+
+<Concept opened>
+## Cold start
+
+Cold start is the time an idle database takes to handle the first request. It usually takes less than five seconds.
+</Concept>
+
+<Concept>
+## Connection pooling
+
+Connection pooling is the process of mutualizing client connections used to connect to the database. By default, connections to PostgreSQL databases uses a lot of memory, which can have a significant impact on performance when hundreds of clients are connected to a database at the same time. With connection pooling, a common component (a "connection pooler", such as PgBouncer) connects to the database and maintains multiple connections to it. Clients then connect to the connection pooler instead of connecting directly to the database, allowing for better performance and more concurrent connections.  
+</Concept>
+
+<Concept>
+## Credentials
+
+Credentials are the login and password (also called "username" and "password" in PostgreSQL vocabulary) used by a user or an application to connect to a database. With Scaleway Serverless SQL Databases, logins are IAM principal IDs and passwords are IAM secret keys.
+</Concept>
+
+<Concept>
+## Engine
+
+A database engine is the software that stores and retrieves your data from a database. Serverless SQL databases use the PostgreSQL engine.
+</Concept>
+
+<Concept>
+## High availability
+
+Databases can become unavailable due to several issues (host crashes, disk corruption, network unavailability, etc.). High Availability allows to ensure that databases stay available most of the time and recover quickly from issues so that applications relying on them remain available as well.
```

Comment:
```suggestion
Databases can become unavailable due to several issues (host crashes, disk corruption, network unavailability, etc.). High Availability ensures that databases stay available most of the time and recover quickly from issues so that applications relying on them remain available as well.
```